index.js

```diff
@@ -0,0 +1,37 @@
+function calculateSwitch(numbers) {
```

Comment:
Since this array has numbers and an operator calling it something like `calculation` would be a better name. Also the function doesn't calculate a switch, it uses a switch but either `performsCalculation` or `calculateAnswer` would be better names.